ports/cpp/test/cpp14/Cpp14Test.cpp

```diff
@@ -38,7 +38,7 @@ TEST(CPP14Parser, SimpleExample) {  // NOLINT: complexity
   CodeCompletionCore completion(&pipeline.parser);
 
   // Ignore operators and the generic ID token.
-  completion.ignoredTokens = {
+  completion.setIgnoredTokens({
```

Comment:
Using a name like `set<anything>` without a `get<anything>` makes this look as if it were write only. Again, I see no need to make this a setter. It's a simple configuration step, why make it complicated? One could pass that in as a paremter to the main collection call, but since this is something configured once for a specific parser, the simple assignment is more appropriate.